students/Wieslaw_Pucilowski/lesson03/Slicing.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Slicing 
+__author___ = "Wieslaw Pucilowski"
+
+### with the first and last items exchanged.
+def exchange_first_last(seq):
+	return seq[-1:]+seq[1:-1]+seq[:1]
```

Comment:
make sure your editor is set to exactly 4 spaces (no tabs)

your editor can do this for you when configured.